python-frontend/src/test/java/org/sonar/python/types/CustomStubSanityTest.java

```diff
@@ -39,6 +40,8 @@
 public class CustomStubSanityTest {
 
   @Test
+  @Ignore
+  //TODO: remove or replace?
```

Comment:
Why is this test ignored? I think this test currently fails because `Typeshed.supportedPythonVersions` is null.
I think we could call `Typeshed.resetBuiltinSymbols()` at the beginning of test and that should make it pass.

Reply:
In fact I think I made my mind to remove, but forgot. This test parses the custom stubs and verifies they're valid. I guess instead it should be removed and something else should do the same in the `Python` code (I thought maybe even just raise some error in the production code if Mypy returns with errors in fact), wdyt?